radio/src/translations/cz.h

```diff
@@ -545,10 +545,10 @@
 #define TR_PRESSANYKEY                 TR("\006Stiskni klávesu", "Stiskni klávesu")
 #define TR_BADEEPROMDATA               TR("\006Chyba dat EEPROM", "Chyba dat EEPROM")
 #define TR_BAD_RADIO_DATA              "Chybná data rádia"
-#define TR_RADIO_DATA_RECOVERED        TR3("Using backup radio data","Using backup radio settings","Radio settings recovered from backup")
-#define TR_RADIO_DATA_UNRECOVERABLE    TR3("Radio settings invalid","Radio settings not valid", "Unable to read valid radio settings")
+#define TR_RADIO_DATA_RECOVERED        TR3("Použití zálohy dat vysílače","Použití zálohy dat vysílače","Nastavení vysílače bylo obnoveno ze zálohy")
+#define TR_RADIO_DATA_UNRECOVERABLE    TR3("Neplatné nastavení vysílače","Neplatné nastavení vysílače", "Nelze načíst platné nastavení vysílače")
```

Comment:
Keep in mind screen size here - there are three different lengths here so it fits 128x64, 212x64 and colorlcd ... so does the first string fit on a 128x64 (i.e. jumper, zorry, tx12) screen? I only ask as I only just managed to make the original `en` text fit ;)

Reply:
I need to test - I have only XD9+ 2019 and TX16 MkII. Anyway I made it shorter just for any case :)

Reply:
If you are able to build the `simu` simulator target you'll be able to test these a lot easier, as you won't need to keep flashing radio (plus can build for radios you don't have). B&W are basically fixed width characters anyway, so once you get the max length down it's just a matter of counting characters.